molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/ExacAnnotator.java

```diff
@@ -25,7 +25,7 @@
 @Configuration
 public class ExacAnnotator
 {
-
+	// TODO Write test
```

Comment:
The old one still has functionality that the new does not, are you sure we need to remove this?